generator/generator.yml

```diff
@@ -720,3 +721,11 @@ modules:
       - 1.3.6.1.4.1.3808.1.1.1 # ups
       - 1.3.6.1.4.1.3808.1.1.4 # environmentSensor
       - 1.3.6.1.4.1.3808.1.1.6 # ePDU2
+#
+# TP LINK EAP
+# http://static.tp-link.com/EAP_Private_Mibs_1.0.zip
+  eap:
+    walk:
+      - interfaces
+      - sysUpTime
```

Comment:
Now that we support multiple modules per scrape, we're deduplicating these from the vendor-specific modules.

You also need to update the `snmp.yml` so the tests pass.

```suggestion
```